### DISCUSSION TITLE
2024年硕博盲评对成果清单的新要求

### DISCUSSION THREAD

**Discussion by YDX-2147483647** [BITNP/BIThesis#495] [Show and tell]
有些学院通知：

> 写于某期刊或者某会议（可以写具体名称），发表论文一篇（不写具体论文名称），作者写：本人第一作者，或者导师第一作者本人第二作者等。

> > 于○○发表论文一篇，本人第一作者。
>
> 多篇按照此格式，前面标序号。

目前模板还没适配，[v3.7](https://github.com/BITNP/BIThesis/releases/tag/v3.7.7) 编译效果如下。

![图片](https://github.com/BITNP/BIThesis/assets/73375426/bdd44e8a-ea1b-443c-b912-3fbceee26b94)

如果您想更改，请参考以下方案。

## 暂时解决方案

盲评时，编辑`misc/4_pub.tex`：

```latex

\begin{publications}
  % 原有内容先注释掉
  % \printbibliography[…]

  % 字号改为五号，编号改用中括号（似乎无明确规定，以下仅供参考）
  \zihao{5}
  \renewcommand{\labelenumi}{[\theenumi]}

  \begin{enumerate}[nosep, leftmargin=4ex-2pt, labelsep=1ex]
    \item 于《新青年》发表论文一篇，本人第一作者。
    \item 于\textit{La Jeunesse}发表论文一篇，导师第一作者，本人第二作者。
  \end{enumerate}
\end{publications}
```

![图片](https://github.com/BITNP/BIThesis/assets/73375426/c21ccae2-0f54-4734-b7fa-c58436d62c7c)

**Comment by fky2015** [comment]
后续我们应该：
1. 在模板中给出混合使用的示例。
2. 或者进一步封装出一个 enumerate 格式。

#515

**Comment by YDX-2147483647** [comment]
为方便大家查看，这个问题归档到了 [🥑 疑难杂症 (bithesis.bitnp.net/faq)](https://bithesis.bitnp.net/faq/)，后续将在那边更新。如果遇到新问题，欢迎继续在 GitHub Discussions 讨论。

https://bithesis.bitnp.net/faq/pub-manual.html
https://github.com/BITNP/BIThesis-wiki/blob/657b4ee88db8d6a48d165965394779bd70432c84/wiki/faq/pub-manual.md
